src/providers/hooks.ts

```diff
@@ -1,16 +1,8 @@
-import { useContext, useMemo } from 'react';
-import { OrganizationContext, UserContext } from './contexts';
+import { useContext } from 'react';
+import { LocalizationContext, OrganizationContext, UserContext } from './contexts';
 
 export const useUser = () => useContext(UserContext);
 
-/**
- * Default time zones until we have a provider.
- */
-export const useTimeZones = () =>
-  useMemo(() => {
-    const defaultTimeZones = (Intl as any).supportedValuesOf('timeZone').map((tz: any) => ({ id: tz, longName: tz }));
-    const supportedTimeZones = [...defaultTimeZones, { id: 'Etc/UTC', longName: 'Coordinated Universal Time' }];
-    return supportedTimeZones;
-  }, []);
+export const useTimeZones = () => useContext(LocalizationContext);
```

Comment:
thoughts on returning the array here, for usage-friendly purposes:
useTimeZones = () => useContext(LocalizationContext).supportedTimeZones;
we will have.. at some point..
useStrings = () => useContext(LocalizationContext).strings;